roles/openshift_master/templates/master.yaml.v1.j2

```diff
@@ -15,8 +15,8 @@ assetConfig:
 {% if 'logging_public_url' in openshift.master %}
   loggingPublicURL: {{ openshift.master.logging_public_url }}
 {% endif %}
-{% if 'metrics_public_url' in openshift.master %}
-  metricsPublicURL: {{ openshift.master.metrics_public_url }}
+{% if openshift_hosted_metrics_deploy | default(false) | bool %}
```

Comment:
In lgith of the other pr maybe this instead?
`{% if openshift_hosted_metrics_public_url | default("") != "" %}`